android-components/components/feature/pwa/src/test/java/mozilla/components/feature/pwa/feature/WebAppContentFeatureTest.kt

```diff
@@ -0,0 +1,61 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.feature.pwa.feature
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import mozilla.components.browser.state.state.BrowserState
+import mozilla.components.browser.state.state.ContentState
+import mozilla.components.browser.state.state.CustomTabConfig
+import mozilla.components.browser.state.state.CustomTabSessionState
+import mozilla.components.browser.state.state.EngineState
+import mozilla.components.browser.state.store.BrowserStore
+import mozilla.components.concept.engine.EngineSession
+import mozilla.components.concept.engine.manifest.WebAppManifest
+import mozilla.components.support.test.mock
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.Mockito.doReturn
+import org.mockito.Mockito.spy
+import org.mockito.Mockito.verify
+
+@RunWith(AndroidJUnit4::class)
+class WebAppContentFeatureTest {
+    private val customTabId = "custom-id"
+
+    @Test
+    fun `display mode is fullscreen based on PWA manifest`() {
+        val engineState: EngineState = mock()
+        val engineSession = mock<EngineSession>()
```

Comment:
```suggestion
        val engineState: EngineState = mock()
        val engineSession: EngineSession = mock()
```

Only a nit: since you started creating a mock using type inference extension above, you could do the same with this one too.